phoenix-core/src/main/java/org/apache/phoenix/hbase/index/Indexer.java

```diff
@@ -225,21 +213,6 @@ public void start(CoprocessorEnvironment e) throws IOException {
         // Metrics impl for the Indexer -- avoiding unnecessary indirection for hadoop-1/2 compat
         this.metricSource = MetricsIndexerSourceFactory.getInstance().create();
         setSlowThresholds(e.getConfiguration());
-
-        try {
-          // get the specified failure policy. We only ever override it in tests, but we need to do it
```

Comment:
Is the goal to no longer have index failure policies because we'll never have (client-visible) index failures?